src/Mono.Android/Mono.Android.projitems

```diff
@@ -140,10 +140,10 @@
       <Stable>True</Stable>
     </AndroidApiInfo>
     <AndroidApiInfo Include="v8.1.99">
```

Comment:
I believe that *this* is the problem: we didn't change it to `v9.0`.  Consequently, when `BuildEverything.mk` builds framework 9.0, `@(_ApiInfo)` is *empty*:

https://github.com/xamarin/xamarin-android/blob/7bd8b7d7748b0734398abf3d966f242ab7eab249/src/Mono.Android/Mono.Android.targets#L138-L141